action.yml

```
@@ -21,3 +21,6 @@ inputs:
     description: If set, will populate the `DEPOT_TOKEN` environment variable.
     default: ''
     required: false
+  target:
+    description: Specific bake target to pull.
+    required: false
```

I believe we do need an explicit default value if `required: false`:

```suggestion
    default: ''
    required: false
```